addons/spreadsheet/static/src/pivot/pivot_runtime.js

```diff
@@ -0,0 +1,137 @@
+/** @odoo-module */
+
+import { _t } from "@web/core/l10n/translation";
+import { PERIODS, parseGroupField } from "@spreadsheet/pivot/pivot_helpers";
+
+class PivotRuntime {
+    constructor(rawDefinition, fields) {
```

Comment:
everywhere in this file
```suggestion
    constructor(definition, fields) {
```